src/main/webapp/WEB-INF/views/account_info.jsp

```diff
@@ -34,10 +34,12 @@
     <span>Overdraft Balance is: $</span><span>${user.overDraftBalance}</span><br/>
     <span>Re-payment logs: </span><span>${user.logs}</span><br/>
     <span>Transaction History: </span><span>${user.transactionHist}</span><br/>
+    <span>Savings: </span><span>${user.savings}</span><br/>
     <br/>
     <a href='/deposit'>Deposit</a>
     <a href='/withdraw'>Withdraw</a>
     <a href='/dispute'>Dispute</a>
+    <a href='/savings'>Savings</a>
```

Comment:
Need to create both a `deposit_savings` and `withdraw_savings` button. It is unintuitive that a "Savings" button goes to deposit savings.

Reply:
You're right, I will add those changes.